examples/castor/amqp_upload.py

```diff
@@ -90,14 +90,14 @@ def parse_sample_line(self, row, line, target_line, separator=','):
             if values[self.csv_config['value_column_idxs'][0]]:
                 data.append({"ts_id"              : values[self.csv_config['sensor_id_idx']],
                              "observed_timestamp": timestamp,
-                             "value"              : float(values[self.csv_config['value_column_idxs'][0]])})
+                             "value"              : values[self.csv_config['value_column_idxs'][0]]})
```

Comment:
You're happy that we don't need to cast this value to a float?

Reply:
Yeah, I introduced this float conversion. It was not in the original code segment in MQTT [upload](https://github.com/IBM/pycloudmessenger/blob/08a8ec7f170fff33022bd2221daa73008a3cb146/pycloudmessenger/castor/mqtt/mqtt_client.py#L102). The conversion throws an error if there is a NaN in the data being committed -- I'm not convinced that that is a bad thing and whether we should allow NaN to be uploaded but I reverted to the original code